python/bigipconfigdriver.py

```diff
@@ -960,23 +1204,36 @@ def main():
         # FIXME (kenr): Big-IP settings are currently static (we ignore any
         #               changes to these fields in subsequent updates). We
         #               may want to make the changes dynamic in the future.
-        bigip = K8sCloudBigIP(host, port,
-                              config['bigip']['username'],
-                              config['bigip']['password'],
-                              config['bigip']['partitions'],
-                              manage_types=[
-                                  '/tm/ltm/virtual',
-                                  '/tm/ltm/pool',
-                                  '/tm/ltm/monitor',
-                                  '/tm/sys/application/service',
-                                  '/tm/ltm/policy'])
-
-        handler = ConfigHandler(args.config_file, bigip, verify_interval)
+
+        # BIG-IP to manage
+        bigip = ManagementRoot(
+            host,
+            config['bigip']['username'],
+            config['bigip']['password'],
+            port=port,
+            token="tmos")
+
+        # Management for the BIG-IP partitions
+        cccls = []
+        for partition in config['bigip']['partitions']:
```

Comment:
This has a risk that's only a problem if someone breaks a contract, but here it is:

We only create cccls for partitions declared the first time we read the config file.  The contract between k8s-bigip-ctlr (go code) and this file is that you can't change that list over time - you have to restart.  However, IMO it's worth verifying that contract is being obeyed since these are two programs in two different languages (there's a fair amount of distance for something to get dropped and this contract to be violated).

We talked about how this risk already exists, and so we decided to fix this later (once we've hoisted a lot of this into the go code).  That's fine with me.